tools/remote_build/rbe_common.bazelrc

```diff
@@ -30,7 +30,6 @@ build --spawn_strategy=remote
 build --strategy=Javac=remote
 build --strategy=Closure=remote
 build --genrule_strategy=remote
-build --remote_timeout=3600
```

Comment:
according to bazel experts, the --remote_timeout flag is unnecessary here (but it falsely was creating the impression that it's important, so I'm removing it).